docs/sources/reference/commandline/cli.md

```diff
@@ -1835,6 +1840,36 @@ An example of a file passed with `--env-file`
 This will create and run a new container with the container name being
 `console`.
 
+    $ sudo docker run -l my-label --label com.example.foo=bar ubuntu bash
```

Comment:
The intro sentence refers to console -- cut-n-paste error.  Below, you meant to refer to the env-file but repeated the ref to label-file.  Tightened presentation

-->
A label is a a `key=value` pair. You can label a container with one or more
labels as follows:

    $ sudo docker run -l my-label --label com.example.foo=bar ubuntu bash

The `my-label` key doesn't specify so the label defaults to an empty
string(`""`). To add multiple labels, repeat the label flag ( `-l` or
`--label`). 

The `key=value` must be unique. If you specify the same `key` multiple times
with different values, each subsequent value overwrites the previous.  Docker
applies the last `key=value` you supply.

Use the `--label-file` flag to load multiple labels from a file. Delimit each
label in the file with an EOL mark. The example below loads labels from a
`labels` file in the current directory;

    $ sudo docker run --label-file ./labels ubuntu bash

The label-file format is similar to the format for loading environment variables
(see `--env-file` above). The following example illustrates a label-file format;

    com.example.label1="a label"

    # this is a comment
    com.example.label2=another\ label
    com.example.label3

You can load multiple label-files by supplying the `--label-file` flag multiple 
times.

For additional information on working with labels, see 
[*Labels - custom meta-data in Docker*](/userguide/labels-custom-metadata/) in
the Docker User Guide.